Tasks/Common/utility-common/kubectlutility.ts

```diff
@@ -109,17 +109,25 @@ function getExecutableExtention(): string {
 
 export async function getAvailableKubectlVersions() {
     var request = new WebRequest();
-    request.uri = "https://api.github.com/repos/kubernetes/kubernetes/releases";
     request.method = "GET";
-
-    try {
-        var response = await sendRequest(request);
-        let versions = [];
-        response.body.forEach(release => {
-            versions.push(release["tag_name"])
-        });
-        return versions;
-    } catch (error) {
-        throw error;
+    let page_number = 0;
```

Comment:
Not related to this PR but below changes avoids calling getAvailableKubectlVersions multiple times. Recently you have changed default version to 1.13.2. Can you change here also https://github.com/Microsoft/azure-pipelines-tasks/blob/f116bb2a9dbcb808d44110fe0ef8f659a57041f4/Tasks/KubernetesV1/src/clusterconnection.ts#L128 and below code path already we are getting version so can you check and remove this line
https://github.com/Microsoft/azure-pipelines-tasks/blob/f116bb2a9dbcb808d44110fe0ef8f659a57041f4/Tasks/KubernetesV1/src/clusterconnection.ts#L135

Reply:
Good catch. I've removed it. 👍